libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Insight/InsightTransposeScenario.tsx

```diff
@@ -15,7 +15,7 @@ const style = { height: 300 };
 
 const InsightViewTransposeCore: React.FC<IPivotTableTransposeCoreProps> = (props) => {
     const { config } = props;
-    const insight = idRef(Insights.TableTranspose);
+    const insight = idRef(Insights.TableWithMC);
```

Comment:
Why you change use this insight without table transpose config in this case?

Reply:
I change it name to more adapt with the test on AD, begin with the Table contains metric + comlumn , then add setting for Table transposition and check

Table transpose will use for KD Table transpose